nemo/collections/tts/losses/uniglowloss.py

```diff
@@ -0,0 +1,187 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The following STFT-loss implementation is taken from:
+# https://github.com/kan-bayashi/ParallelWaveGAN/blob/master/parallel_wavegan/losses/stft_loss.py
```

Comment:
Can you add their copyright header here? Like it is done in https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/tts/modules/squeezewave.py

Reply:
Can you copy in the whole license?

Reply:
Actually, regarding my email, did you modify the code from parallel_wavegan at all? If not, we should just use them as a pip dependency